### PR TITLE
chore(ci): Install latest yarn via apt-get

### PR DESCRIPTION
**Pull request by rwjblue**
Based on instructions from:

https://yarnpkg.com/lang/en/docs/install-ci/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,12 +6,17 @@ node_js:
...
 cache:
   yarn: true
 
 
+before_install:
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y -qq yarn
+  - yarn --version
```
Comment (Turbo87) [inline .travis.yml]: I would suggest to use the same steps as in all the other repos:
https://github.com/ember-cli/ember-cli/blob/453c105ef4a15feb3ed1e7c26dd81e835d15095a/tests/fixtures/app/yarn/.travis.yml#L21-L22

Comment (rwjblue) [reply .travis.yml]: yeah, I ended up doing that

Comment (rwjblue) [reply .travis.yml]: but its dumb that the thing https://yarnpkg.com/en/docs/install-ci says doesn't work 😠

Comment (Turbo87) [reply .travis.yml]: I totally agree. I'm also not sure why they recommend using `sudo` commands when there is clearly no need for it...

**Pull request merged**